src/App.jsx

```diff
@@ -1,83 +1,73 @@
 import 'bulma/css/bulma.css';
 import './App.scss';
+import { useState } from 'react';
 
-export const goods = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
-];
+export const App = () => {
+  const goods = [
+    'Dumplings',
+    'Carrot',
+    'Eggs',
+    'Ice cream',
+    'Apple',
+    'Bread',
+    'Fish',
+    'Honey',
+    'Jam',
+    'Garlic',
+  ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const handleGoodSelect = (good) => {
+    if (selectedGood === good) {
+      setSelectedGood('');
+    } else {
+      setSelectedGood(good);
+    }
+  };
```

Comment:
```suggestion
  const handleGoodSelect = (good) => () => {
      setSelectedGood(good);
  };
```
suggest to decide in jsx what we wil be set and return arrow function. it will prevent to make anonymous arrow function in jsx.

Reply:
I changed it and buttons ('+' and '-') stopped working

Reply:
Are you checked all comments?
This [comment](https://github.com/mate-academy/react_goods-selector-js/pull/389/files#r1359447360) connected with changes in handleGoodSelect function.